source/locale/pt_BR/LC_MESSAGES/faq.po

```diff
@@ -99,11 +96,11 @@ msgstr "Onde eu posso baixar?"
 
 #: ../../source/faq/faq.rst:39
 msgid ""
-"All components are available in dojot's GitHub repositories: "
-"`<https://github.com/dojot>`_."
+"All components are available in dojot's GitHub repositories: `<https://github."
+"com/dojot>`_."
```

Comment:
The link is separated in here, could this break the link in the documentation too?

Reply:
No, it works well :)